src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
Pay attention to access modifiers. The main method is correctly public, but ensure that all other classes, fields, constructors, methods have access modifiers as well.